apps/web/src/components/layout/components/EnsureOnboardingComplete.tsx

```diff
@@ -6,9 +6,14 @@ import { useBlueprint } from '../../../hooks/index';
 export function EnsureOnboardingComplete({ children }: any) {
   useBlueprint();
   const location = useLocation();
-  const { currentOrganization, environmentId } = useAuth();
+  const { currentUser, currentOrganization } = useAuth();
 
-  if ((!currentOrganization || !environmentId) && location.pathname !== ROUTES.AUTH_APPLICATION) {
+  if (!currentUser) {
```

Comment:
Minor auth fix, in case something goes wrong with the fetch API requests, go to sign-in not /auth/application